lib/util_quda.cpp

```diff
@@ -85,7 +85,7 @@ void pushVerbosity(QudaVerbosity verbosity)
   vstack.push(getVerbosity());
   setVerbosity(verbosity);
 
-  if (vstack.size() > 10) {
+  if (vstack.size() > 15) {
```

Comment:
Curious why this change was needed?

Reply:
I never did a super-explicit trace and counting, but doing a four-level solve blew out a size of 10, while 15 seemed to be enough. The `Level 1`, etc, prints during the setup and solve always made consistent sense, so it wasn't actually because of a `popVerbosity/OutputPrefix` mismatch.

Reply:
I don't deny that it's surprising 10 wasn't sufficient.